frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx

```diff
@@ -169,6 +182,7 @@ export const FieldRowContainer = ({
             align="center"
             onClick={handleFieldClick}
             onKeyDown={handleKeydown}
+            ref={isActive ? activeFieldRef : undefined}
```

Comment:
is there a reason why `ref` is only assigned on isActive state? We can also just assign the ref regardless, and call scrollIntoView when active.

Reply:
sure! 992ae61c